cmd/appengine/device_aliases.go

```diff
@@ -55,6 +57,7 @@ var aliasesRemoveCmd = &cobra.Command{
 	Example: `  astartectl appengine devices aliases remove 2TBn-jNESuuHamE2Zo1anA my-alias-tag`,
 	Args:    cobra.ExactArgs(2),
 	RunE:    aliasesDeleteF,
```

Comment:
I think we should rename `aliasesDeleteF` to `aliasesRemoveF`

Reply:
++